code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm

```diff
@@ -55,10 +55,10 @@
 /mob/living/simple_animal/hostile/megafauna/ex_act(severity, target)
 	switch (severity)
 		if (1)
-			adjustBruteLoss(250)
+			adjustBruteLoss(400)
```

Comment:
I already said it more than once, and I say it again. Using bombs and risking getting caught in explosion for a chance to deal 50 damage is simply not worth it when you can just shot the boss with KA twice for 80 damage with zero risk.